tests/test_fourier.py

```diff
@@ -95,16 +96,53 @@ def test_make_fourier_transform():
 	ft = make_fourier_transform(input_grid, q=1, fov=1, planner='estimate')
 	assert type(ft) == FastFourierTransform
 
+	fft_grid = make_fft_grid(input_grid, q=1, fov=1)
+	ft = make_fourier_transform(input_grid, fft_grid, planner='estimate')
+	assert type(ft) == FastFourierTransform
+
 	ft = make_fourier_transform(input_grid, q=8, fov=0.3, planner='estimate')
 	assert type(ft) == MatrixFourierTransform
 
+	fft_grid = make_fft_grid(input_grid, q=8, fov=0.3)
+	ft = make_fourier_transform(input_grid, fft_grid, planner='estimate')
+	assert type(ft) == MatrixFourierTransform
+
 	ft = make_fourier_transform(input_grid, q=1, fov=1, planner='measure')
 	ft = make_fourier_transform(input_grid, q=8, fov=0.1, planner='measure')
 
 	output_grid = CartesianGrid(UnstructuredCoords([np.random.randn(100), np.random.randn(100)]))
 	ft = make_fourier_transform(input_grid, output_grid)
 	assert type(ft) == NaiveFourierTransform
 
+def test_fft_grid_reconstruction():
+	for shift_input in [[0,0], [0.1]]:
```

Comment:
spaces on lines 118, 120 and 123.

Reply:
See previous comment. I tried to keep the pairs together, for visual reasons, but I'm fine with using spaces for all. Let me know which you think is cleaner.